README.md

```diff
@@ -36,12 +36,6 @@ For testing you can do it with the command:
 
 - npm start
```

Comment:
- [ ] This command is for running the app not the test. Please change `Run tests` to `Usage` and add another step `npm install`